src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImportWithBlockMethodParameters.java

```diff
@@ -0,0 +1,15 @@
+package com.puppycrawl.tools.checkstyle.checks.imports.unusedimports;
+
+import java.util.Collection;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.TimeUnit;
+
+public class InputUnusedImportWithBlockMethodParameters {
+
+/**
+ * @see ExecutorService#invokeAll(Collection, long, TimeUnit)
```

Comment:
If you prefer, I could just add this `@see` tag to one of the javadoc blocks in `InputUnusedImports.java`. If I did that, there wouldn't need to be any changes to `UnusedImportsCheckTest.java`.